content/workload-reduction-toolkit/address-workload-issues/feedback-and-marking/cut-down-your-marking-workload.md

```diff
@@ -0,0 +1,90 @@
+---
+
+# Cut down your marking workload
+
+<strong class="govuk-tag">Case study</strong>
+
+{inset-text}
+
+## School details
+
+**School name:** Barr Beacon School
+
+**Location:** Walsall, West Midlands
+
+**Phase:** Secondary
+
+**Number of pupils:** 1500
+
+**Contact details:** Email Deputy Headteacher David Lowbridge-Ellis at <dlowbridgeellis@matrixacademytrust.co.uk> 
+
+{/inset-text}
+
+<div class="govuk-grid-row dfe-width-container">
+  <div class="govuk-grid-column-full">
+    <div class="info-box">
+      <div class="info-box__corner">
+        <img src="/assets/images/bullseye.svg" alt="Bullseye icon">
+      </div>
+      <h2 class="govuk-heading-m">
+        Impact and outcomes
+      </h2>
+      <p>
+        Within a term of encouraging teachers to try alternative approaches, some teachers said their marking workload had reduced by up to 75%. Several teachers said they “couldn’t believe we used to spend four hours on a set of exercise books. Now I spend much less time marking.”  
+      </p>
+      <p>
+      To make sure marking was contributing to pupil progress, we rigorously monitored the quality of feedback by asking pupils “what are you doing well in this subject?” and “what do you need to do to improve in this subject?”
+      </p>
+    </div>
+  </div>
+</div>
+
+## Background from David Lowbridge-Ellis, Deputy Headteacher
+
+{inset-text}
+I asked staff to mark however they like. This was so they could try new approaches which were more specific to their subjects and the pupils in front of them. I collated some of these ideas in a document and sent it to all staff to further encourage them to try doing things in a more time efficient manner. 
+{/inset-text}
+
+## Alternatives to written diagnostic feedback 
+
+There will be times where it is appropriate to take in pupils’ work and write diagnostic feedback on it. Often, there is a more time-efficient method of giving pupils feedback, depending on subject and class size.  
+
+In our school, we have found these suggestions to help reduce teacher’s workload when it comes to marking.  
+
+### Verbal feedback may be quicker than written feedback 
+
+Live writing on the board – the teacher models how to write the answer, giving a commentary of the thoughts that run through their head as they do, including making mistakes and corrections (English, Science). 
+
+With sixth formers, record an audio file of you reading and commenting on a piece of work. This can be sent to the pupil to listen back (Media, Film). 
+
+### Standardising what feedback says reduces time spent writing repetitive feedback 
+
+Use standardised codes (Languages, English, Humanities). 
+
+Generate automatic feedback comments off exam spreadsheets (Science, Computing, PE). 
+
+When diagnostic assessment is most useful, use a diagnostic grid and select boxes rather than write sentences. 
+
+ ### Enable self or peer marking amongst students through the choice of assignment and by providing marking criteria 
```

Comment:
Extra space before # will break the markdown:
<img width="741" alt="Screenshot 2024-02-20 at 10 08 29" src="https://github.com/DFE-Digital/improve-school-workload-and-wellbeing/assets/18436946/da87c67d-c25e-45a8-9c23-a399c48b9816">